src/commands/new.ts

```diff
@@ -33,12 +33,9 @@ export default {
     if (parameters.options.overwrite) {
       remove(projectName)
     } else if (exists(projectName)) {
+      const alreadyExists = `Error: There's already a folder with the name ${projectName}. To force overwriting that folder, run with --overwrite.`
```

Comment:
Do we still want double quotes around the name or not?

Reply:
Agreed.  Do quotes.